forge-gui/res/cardsfolder/upcoming/tectonic_split.txt

```diff
@@ -0,0 +1,9 @@
+Name:Tectonic Split
+ManaCost:4 G G
+Types:Enchantment
+K:AlternateAdditionalCost:Sac<X/Land>
```

Comment:
wrong KW usage

Reply:
<img width="1131" height="384" alt="grafik" src="https://github.com/user-attachments/assets/2494ef1d-d850-41e6-943a-8e8a0acc2bbb" />

Reply:
it's older approach